examples/image_classification/README.md

```diff
@@ -1,4 +1,22 @@
-# Fine-tuning for image classification using LoRA and 🤗 PEFT
+1. Setup env - train all models
+```
+sh run_all_models.sh model_batchsize.txt
+```
+* To track top GPU vRAM usage by device, open a new terminal and run this script, pass device ID, for ex device 2:
+```
+sh ../memory_record_moreh.sh 2
```

Comment:
There are still some `sh` here. Please make sure you've changed all the README files for each task.